src/api/models/Activity.ts

```diff
@@ -16,7 +16,7 @@ import {
     HasManyGetAssociationsMixin,
     HasManyHasAssociationMixin,
     HasManyRemoveAssociationMixin,
-    HasManyRemoveAssociationsMixin,
+    HasManyRemoveAssociationsMixin, HasOneCreateAssociationMixin, HasOneGetAssociationMixin, HasOneSetAssociationMixin,
```

Comment:
Since all the other are on their own line, you should split this line on multiple lines